packages/hooks/src/useCurrencyExchangeRate.ts

```diff
@@ -0,0 +1,10 @@
+import { useStore } from '@deriv/stores';
+
+const useCurrencyExchangeRate = (currency: string) => {
+    const { exchange_rates } = useStore();
+    const rate = exchange_rates.data?.rates?.[currency || 'USD'] || 1;
```

Comment:
```suggestion
    const rate = exchange_rates.data?.rates?.[currency] || 1;
```
@shayan-deriv The `currency` can't be `undefined`.